pedsnetdcc/tests/db_test.py

```diff
@@ -0,0 +1,484 @@
+import json
+import logging
+import multiprocessing
+import psycopg2
+import testing.postgresql
```

Comment:
Yeah, it seems to work pretty well. I occasionally get an error as the tests are ending, which I think is due to some sort of garbage collection race condition. It's also pretty speedy. The test time went up to about 11 seconds, IIRC, but that's not bad. We can always move the db-dependent tests into the outer "integration testing" tests directory if we want to be able to run the speedy unit tests separately.

Also, since it uses temp directories and files, it's a place I would expect might need some attention if/when we move to a CI solution.